tests/python/unittest/test_operator.py

```diff
@@ -367,6 +367,108 @@ def test_pow_fn():
     check_symbolic_forward(y, [x], [2**x])
     check_symbolic_backward(y, [x], [np.ones(shape)], [np.log(2) * 2**x])
 
+def test_equal():
```

Comment:
Do you know what lint version is used(or any pointers to the documentation) on the the test server? I couldn't find it on mxnet.io. It is hard to fix issues which are not locally reproducible. Maybe we should consider running lint as the part of the regular make(including checking the appropriate version and logging it if it is not) so that we catch them early.